src/Transformer/DenormalizationTransformerManager.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flagbit\Bundle\CategoryBundle\Transformer;
+
+use Flagbit\Bundle\CategoryBundle\Entity\CategoryConfig;
+use Flagbit\Bundle\CategoryBundle\Repository\CategoryConfigRepository;
+
+/**
+ * Handle property denormalization transformer registration and execution.
+ *
+ * This class handles the registration and execution of denormalization transformers, that allow
+ * to apply custom logic for specific property types during denormalization.
+ *
+ * Symfony service tags are used to register new transformers.
+ * The tag used to register a service as a denormalization transformer
+ * is "flagbit.category.data_transformer.denormalization".
+ * A "type" attribute must be passed with the tag, that defines on which property type the
+ * transformer should be applied.
+ *
+ * Transformers must implement the {@see DenormalizationTransformer} interface.
+ */
+class DenormalizationTransformerManager
+{
+    public const DEFAULT_NORMALIZATION_TRANSFORMER   = 'default';
+    private const DEFAULT_PROPERTY_CONFIG_IDENTIFIER = 1;
```

Comment:
If this is the id of the database row, then you can actually create a repository method to hide the id there. The `DenormalizationTransformerManager` is not supposed to know about such things.